### DISCUSSION TITLE
(password) protect preferences page

### DISCUSSION THREAD

**Discussion by Panja0** [searx/searx#3095] [Installation issue: Docker]
I'm using Searx and NGinx Proxy Manager on Docker. Everything works fine except I would like to keep the preferences page "private".
I will need to have it password protected.

Is there a way to do this?

**Comment by unixfox** [comment]
Just put an auth on the /preferences page

**Comment by Panja0** [comment]
I can't seem to get that working with NPM...

**Reply by unixfox** [reply]
you should open an issue on NPM then, it's a github repository for Searx not NPM.

**Reply by Panja0** [reply]
Yes, I understand. I was hoping there was an option in Searx itself to secure the preferences page.
Sorry for taking your time.
